### DISCUSSION TITLE
Mount existing folders  with files to minio

### DISCUSSION THREAD

**Discussion by justlucknb** [minio/minio#15496] [Q&A] [answered]
Hello, i run docker-compose and share volumes with files to minio contanier. 
Then when i connect to aws, my bucket is empty. How mount existing files to minio?

**Comment by harshavardhana** [comment, marked as answer]
Existing folder support has been removed in last 3 months.

**Reply by justlucknb** [reply]
Ok, now i used [RELEASE.2021-11-03T03-36-36Z but my folder also empty.

**Reply by ravindk89** [reply]
When you run MinIO against a file path, it creates backend metadata within that path.

Simply downgrading the release without completely emptying the folder - including the `.minio.sys` folder - probably wont work.

Try to start with a completely fresh folder, then use a release prior to `2022-06-02` - that should show the deprecated FS-style behavior.

The better thing to do is to start your docker containers and run a script that uses `mc cp` or `mc mv` to populate the MinIO folder with your pre-existing files. This lets you take advantage of the features provided by the single-node single-drive backend like versioning and object locking.

**Comment by audouts** [comment]
@harshavardhana Why was this removed? My MinIO drive has a large amount of data on it and I would like to use that data in MinIO. Is there no way to do that without this procedure?

**Reply by audouts** [reply]
I tried `mc mv` and that didn't work. The files got moved but they don't exist in the MinIO console object browser.

EDIT:
I didn't know how to use `mv` to the server - it was using the local filesystem. That was fast but didn't properly add the files. Now I'm using http (localhost). This is slow but I guess that's the only way to do it.

**Reply by ravindk89** [reply]
> That was fast but didn't properly add the files. 

You aren't giving critical details that would assist in troubleshooting.

How did you deploy MinIO? What version? What command(s) are you running, and what were you expecting the output to be? what was the result actually?

**Reply by audouts** [reply]
Hi @ravindk89 and thanks for your reply.

I'm using the latest version via Docker. As I mentioned above, I was trying to use `mc mv` to get files into the bucket but I was copying them across the same, local filesystem. I had hoped that would "import" the files but I see that method doesn't create any metadata or process the files.

I had to create an alias and move it to the bucket inside that alias.

**Reply by harshavardhana** [reply]
It is a product design decision.

**Comment by viniciusregitano** [comment]
I'm also facing the same issue here.
Will this feature ever come back to MinIO?

**Reply by klauspost** [reply]
No.
